leetcodePython/BinarySearch/bs_1648.py

```diff
@@ -0,0 +1,25 @@
+import heapq
+from typing import List
+
+
+class Solution:
+    def arithmeticSum(self, a_n: int, a_1: int, n: int) -> int:
+        return (a_n + a_1 + 1) * n // 2
```

Comment:
`[6, 4, 2]`

We can get value:
`6, 5` -> `[4, 4, 2]`

The diff value sum `6, 5` becomes:
`(6 + 4 + 1) * (6 - 4) / 2`